internal/local/hosting.go

```diff
@@ -208,10 +207,12 @@ func (h Hosting) UploadHostingAssets(realmClient realm.Client, groupID, appID st
 		}()
 	}
 
+	assetsDir := filepath.Join(h.RootDir, NameFiles)
```

Comment:
cc: @andrew this is where can we ensure we use the right file path when uploading assets (we have a `const` for this `files/` path part in the `local` package ... can't use that in `realm` package though)

Reply:
This works for me!